### PR TITLE
Be consistent in code examples

### PR DESCRIPTION
**Pull request by OskarStark**
Follows https://github.com/symfony/symfony-docs/pull/15731#discussion_r1133067695

cc @HeahDude

### CODE REVIEW

File: service_container/service_closures.rst [hunk 1]
```diff
@@ -69,6 +72,11 @@ argument of type ``service_closure``:
             <services>
                 <service id="App\Service\MyService">
                     <argument type="service_closure" id="mailer"/>
+
+                    <!--
+                    In case the dependency is optional
+                    <argument type="service" id="logger" on-invalid="ignore"/>
```
Comment (OskarStark) [inline service_container/service_closures.rst]:
```suggestion
                    <argument type="service" id="mailer" on-invalid="ignore"/>
```

File: service_container/service_closures.rst [hunk 2]
```diff
@@ -58,6 +58,9 @@ argument of type ``service_closure``:
             App\Service\MyService:
                 arguments: [!service_closure '@mailer']
 
+                # In case the dependency is optional
+                # arguments: [!service_closure '@?mailer']
```
Comment (HeahDude) [inline service_container/service_closures.rst:62]: I'm not sure if this works actually

File: service_container/service_closures.rst [hunk 3]
```diff
@@ -69,6 +72,11 @@ argument of type ``service_closure``:
             <services>
                 <service id="App\Service\MyService">
                     <argument type="service_closure" id="mailer"/>
+
+                    <!--
+                    In case the dependency is optional
+                    <argument type="service" id="mailer" on-invalid="ignore"/>
```
Comment (HeahDude) [inline service_container/service_closures.rst]:
```suggestion
                    <argument type="service_closure" id="mailer" on-invalid="ignore"/>
```

**Comment by OskarStark** [comment]
@nicolas-grekas can you confirm?

File: service_container/service_closures.rst (hunk 2 above)
Comment (alamirault) [reply service_container/service_closures.rst:62]: I checked and it's ok, if `mailer` service doesn't exist, null is pass instead

**Review by alamirault — APPROVED** [review]

**Review by nicolas-grekas — APPROVED** [review]

**Pull request merged**